images/openshift-golang-builder.yml

```diff
@@ -7,6 +7,15 @@ content:
       branch:
         target: rhel-9-golang-1.20
       url: git@github.com:openshift/ocp-build-data.git
+    modifications:
+    - action: replace
+      match: "# FOD wrapper modification"
+      replacement: "COPY go_wrapper.sh /tmp/go_wrapper.sh
+RUN /bin/bash -c 'GO_BIN_PATH=`which go`; mv $GO_BIN_PATH $GO_BIN_PATH.real; mv /tmp/go_wrapper.sh $GO_BIN_PATH; chmod +x $GO_BIN_PATH'"
+    - action: add
```

Comment:
`doozer` is running the rebase code, so would have access to the shell file of the current commit. Think it'd be nicer to not add a network call to `master`, but to load the file while rebasing.